assets/base/i18n/en.json

```diff
@@ -1177,6 +1177,7 @@
   "emoji_list.name": "Name",
   "emoji_list.search": "Search Custom Emoji",
   "emoji_list.somebody": "Somebody on another team",
+  "error.network_request_failed": "Cannot connect to the server. Please check your server URL and internet connection.",
```

Comment:
What if we just threw the FormattedError by default in the catch block? The only operation in the try block is the ```Client.getPing()``` so if an error occurs it will most likely be related to a fetch. As for the id maybe use ```mobile.routes.select_server.server_response_error```?